### ISSUE TITLE
Rename all of the strings?

### ISSUE THREAD

**Issue by iopq** [xbclub/xraya#1]
Should all instances of v2ray be replaced with xray?

I got this to work, but if I'm going to package it it would be awkward to use a lot of v2ray named stuff

**Comment by xbclub** [comment]
The current version only modifies v2raya to support the xtl rprx-vision flow control for xray. After the official release of the xray reality protocol, we will consider a complete refactoring

**Comment by xbclub** [comment]
Of course, if you can help us with string renaming, we welcome pull requests

**Comment by iopq** [comment]
I just did a pull request for the readme!
